torchx/nerves/toolchain.cmake

```diff
@@ -0,0 +1,28 @@
+# maybe the easiest way to identify a nerves build?
+if(NOT "$ENV{TARGET_CPU}" STREQUAL "" AND NOT "$ENV{TARGET_ARCH}" STREQUAL "" AND NOT "$ENV{TARGET_OS}" STREQUAL "" AND NOT "$ENV{TARGET_ABI}" STREQUAL "")
```

Comment:
I'll wait on the others to respond on whether we should include Nerves-specific things in Nx. Perhaps we're gonna need another solution for people to have this available without actually incorporating this in the codebase.

Reply:
My main concern is: if we're willing to add Nerves-specific code to Nx, why wouldn't we support X or Y library  with more specific code? Also, changes on the Nerves side could break this code section as well.